web/cypress/e2e/TestRunDetail/NewFlows.spec.ts

```diff
@@ -0,0 +1,60 @@
+import {getAttributeListId} from '../../support/commands';
+
```

Comment:
Instead of just naming it new flows, lets have a separate name for each of them. Because once we add more flows these are going to be come "new old flows" 😂